doc/rst/technotes/gpu.rst

```diff
@@ -139,22 +139,27 @@ To enable GPU support set the environment variable ``CHPL_LOCALE_MODEL=gpu``
 before building Chapel.
 
 Chapel's build system will automatically try and deduce what type of GPU you
-have and where your installation of relevant runtime (e.g. CUDA or ROCM) are.
-If the type of GPU is not detected you may set ``CHPL_GPU_CODEGEN`` manually to
-either ``cuda`` (for NVIDIA GPUs) or ``rocm`` (for AMD GPUs). If the relevant
-runtime path is not automatically detected (or you would like to use a
-different installation) you may set ``CHPL_CUDA_PATH`` and/or
-``CHPL_ROCM_PATH``.
+have and where your installation of relevant runtime (e.g. CUDA or ROCm) are.
+If the type of GPU is not detected you may set ``CHPL_GPU`` manually to either
+``nvidia`` or ``amd`` . If the relevant runtime path is not automatically
+detected (or you would like to use a different installation) you may set
+``CHPL_CUDA_PATH`` and/or ``CHPL_ROCM_PATH``.
 
 ``CHPL_GPU_ARCH`` environment variable can be set to control the desired GPU
 architecture to compile for.  The default value is ``sm_60`` for
-``CHPL_GPU_CODEGEN=cuda`` and ``gfx906`` for ``CHPL_GPU_CODEGEN=rocm``. You may
-also use the ``--gpu-arch`` compiler flag to set GPU architecture. For a list
-of possible values please refer to `CUDA Programming Guide
+``CHPL_GPU=nvidia`` and ``gfx906`` for ``CHPL_GPU=amd``. You may also use the
+``--gpu-arch`` compiler flag to set GPU architecture. For a list of possible
+values please refer to `CUDA Programming Guide
 <https://docs.nvidia.com/cuda/cuda-c-programming-guide/#features-and-technical-specifications>`_
 for NVIDIA or "processor" values in `this table in the LLVM documentation
 <https://llvm.org/docs/AMDGPUUsage.html#processors>`_ for AMD.
 
+``CHPL_RT_NUM_GPUS_PER_LOCALE`` can be used to control the number of GPU
+sublocales created. For example, in a system where 8 physical GPUs per node,
+setting this environment variable to 4 will make Chapel use only the first 4
+GPUs. Setting this number to a value higher than the number of GPUs per node
+results in an error.
```

Comment:
Add: When `CHPL_GPU=cpu`, you may set `CHPL_RT_NUM_GPUS_PER_LOCALE` to any non-negative number.